Lib/http/server.py

```diff
@@ -1251,6 +1258,42 @@ def run_cgi(self):
                 self.log_message("CGI script exited OK")
 
 
+class HTTPSServer(HTTPServer):
+    def __init__(self, server_address, RequestHandlerClass,
+                 bind_and_activate=True, *, certfile, keyfile=None,
+                 password=None, alpn_protocols=None):
+        if ssl is None:
+            raise ImportError("SSL support missing")
```

Comment:
Is it the right exception to raise? shouldn't we raise something else? maybe a RuntimeError? (it feels weird to raise an import error in the constructor)